lib/pagseguro/sender.rb

```diff
@@ -15,6 +15,10 @@ class Sender
     # Set the CPF document.
     attr_accessor :cpf
 
+    # Set sender hash.
+    # It's used to identify the sender.
+    attr_accessor :hash
```

Comment:
to be honest, I don't like this name neither. just following the api attribute name.